Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -107,6 +107,14 @@ private BoolQueryBuilder createUserTaskQuery(User user) {
         query.must(createSimpleQuery("processingStatus", TaskStatus.LOCKED.getValue(), false));
         query.must(createSimpleQuery("processingStatus", TaskStatus.DONE.getValue(), false));
 
+        List<JSONObject> templateProcesses = serviceManager.getTemplateService().findByTemplate(true, null);
```

Comment:
Following code is already existing in method `getAmountOfCurrentTasks`. Instead of duplicating the code, the code should be moved in a separate method and called from both methods.